lib/fathead/scala/parse.py

```diff
@@ -13,6 +13,23 @@ def get_all_files():
 			doc_files.append( "%s/%s" % (path, f))
 	return doc_files
 
+def extarct_article_text(article):
+	article_text = ""    
+	for i in article.children:
+        	if i.name == "pre" or i.name == "code":
+	        	article_text += "<pre>" + extarct_article_text(i) + "</pre>"
```

Comment:
@moollaza That would mean that when a `code` appears inside a paragraph it should be inline and otherwise it should break the page.